plugin/advisor/utils_for_tests.go

```diff
@@ -12,58 +10,120 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-// MockCatalogService is the mock catalog service for test.
-type MockCatalogService struct{}
-
 const (
 	// MockOldIndexName is the mock old index for test.
 	MockOldIndexName = "old_index"
 	// MockOldUKName is the mock old unique key for test.
 	MockOldUKName = "old_uk"
-	// MockOldPKName is the mock old foreign key for test.
-	MockOldPKName = "PRIMARY"
+	// MockOldMySQLPKName is the mock old foreign key for test.
```

Comment:
the comment doesn't seem to agree with the value

Reply:
aha, let me convince them.